### ISSUE TITLE
Accessibility Issues on CC Homepage: post project-gen

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2224]
![image](https://cloud.githubusercontent.com/assets/5177491/23570278/5baa819a-0018-11e7-96f7-38ee0adac9ac.png)

**Comment by eclectir** [comment]
Error details-
MissingProperty - UIA_NameProperty - Element has no accessible name. If a user reaches an element with no name, then often they will have no way of knowing what the element relates to.
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.automationelementinformation.name(v=vs.110).aspx
